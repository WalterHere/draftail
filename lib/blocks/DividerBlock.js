import React from 'react';

/**
 * The equivalent of an <hr/> in the editor.
 */
const DividerBlock = () => (
    <div className="RichEditor-horizontal-rule" />
);

export default DividerBlock;
