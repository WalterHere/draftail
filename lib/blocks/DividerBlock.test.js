import React from 'react';
import { shallow } from 'enzyme';
import DividerBlock from '../blocks/DividerBlock';

describe('DividerBlock', () => {
    it('exists', () => {
        expect(DividerBlock).toBeDefined();
    });

    it('basic', () => {
        expect(shallow(<DividerBlock />)).toBeDefined();
    });
});
