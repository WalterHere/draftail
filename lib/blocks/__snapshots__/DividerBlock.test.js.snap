exports[`DividerBlock basic 1`] = `<hr />`;
